src/pages/decks/index.tsx

```diff
@@ -54,12 +54,14 @@ export default function Home() {
     }
   }
 
-  if (isLoading || !data) {
```

Comment:
Did `!data` move somewhere else/was it already somewhere else?

Reply:
Good q!! Made me finally look at the documentation https://react-query.tanstack.com/guides/queries "For most queries, it's usually sufficient to check for the isLoading state, then the isError state, then finally, assume that the data is available and render the successful state" so if I'm reading correctly, we don't necessary need to check for !data, but we might want to have an isError check (I believe right now it will just show nothing if there's an error)